js/common/grids.d.ts

```diff
@@ -570,6 +570,39 @@ export type ColumnFixingTexts = {
   unfix?: string;
 };
 
+/**
+ * @public
+ * @namespace DevExpress.common.grids
+ */
+export type ColumnHeaderFilterSearch = {
+  /**
+   * @docid GridBaseColumn.headerFilter.search.editorOptions
+   * @default {}
+   */
+  editorOptions?: any;
+  /**
+   * @docid GridBaseColumn.headerFilter.search.enabled
+   * @default false
+   */
+  enabled?: boolean;
+  /**
+   * @docid GridBaseColumn.headerFilter.search.mode
+   * @default 'contains'
+   */
+  mode?: SearchMode;
+  /**
+   * @docid GridBaseColumn.headerFilter.search.searchExpr
+   * @type getter|Array<getter>
+   * @default undefined
+   */
+  searchExpr?: string | Function | Array<string | Function>;
+  /**
+   * @docid GridBaseColumn.headerFilter.search.timeout
+   * @default 500
+   */
+  timeout?: number;
+};
```

Comment:
I agree that this type should be derived from HeaderFilterSearch. And, since the types are new, we can leave the docid tags empty. Will we have some problems with documentation in this case?

Reply:
<img width="1165" alt="image" src="https://user-images.githubusercontent.com/84278206/227085644-ef8cac3e-882b-4e50-830b-e58c315d48f8.png">

that's how it looks with empty docids in dx.all.d.ts. Is it ok?